lcu_driver/utils.py

```diff
@@ -13,9 +13,18 @@ def parse_cmdline_args(cmdline_args) -> Dict[str, str]:
 
 
 def _return_ux_process() -> Generator[Process, None, None]:
-    for process in process_iter():
-        if process.status() == STATUS_ZOMBIE:
-            continue
+    for process in process_iter(attrs=["cmdline"]):
+        try:
```

Comment:
What's the purpose of this try/catch? If such errors occur we should know to fix them (e.g. [known problems](https://github.com/sousa-andre/lcu-driver/tree/master#problems)) and not hide them away

Reply:
I tested without it and got no problems. So i'd say it's probably safe to have it like it was before.  (Meaning no try:catch block.)